src/components/DocumentationTopic/DocumentationHero.vue

```diff
@@ -153,6 +161,11 @@ $doc-hero-icon-dimension: 250px;
   }
 }
 
+.extra-padding {
+  padding-top: rem(60px);
+  padding-bottom: 60px;
```

Comment:
Make the bottom rem as well 🤔 ?

```suggestion
  padding-bottom: rem(60px);
```